CHANGELOG.md

```diff
@@ -1,8 +1,87 @@
 # Changelog
 
 ## [1.41.1](https://github.com/brave/brave-ios/releases/tag/v1.41.1)
+
 - Fixed usage ping from reporting week of install as Sunday instead of Monday. ([#5750](https://github.com/brave/brave-ios/issues/5750))
 
+## [1.41](https://github.com/brave/brave-ios/releases/tag/v1.41)
+
+- Updated Ecosia search engine logo under search engine settings. ([#5504](https://github.com/brave/brave-ios/issues/5504))
+- Updated URL bar and bottom toolbar transition during page scroll. ([#5645](https://github.com/brave/brave-ios/issues/5645))
+- Fixed 'Try Brave Search' button to show full text on multi-line for certain languages. ([#5644](https://github.com/brave/brave-ios/issues/5644))
```

Comment:
Remove the single quotes or GH is going to format `Try Brave Search'. Should appear as "Try Brave Search"